tests/network/vmi_istio.go

```diff
@@ -489,15 +489,15 @@ func newVMIWithIstioSidecar(ports []v1.Port, vmType VmType) (*v1.VirtualMachineI
 }
 
 func createMasqueradeVm(ports []v1.Port) (*v1.VirtualMachineInstance, error) {
-	networkData, err := libnet.CreateDefaultCloudInitNetworkData()
+	networkData := libnet.CreateDefaultCloudInitNetworkData()
 	vmi := libvmi.NewAlpineWithTestTooling(
 		libvmi.WithNetwork(v1.DefaultPodNetwork()),
 		libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding(ports...)),
 		libvmi.WithLabel(vmiAppSelectorKey, vmiAppSelectorValue),
 		libvmi.WithAnnotation(istio.ISTIO_INJECT_ANNOTATION, "true"),
 		libvmi.WithCloudInitNoCloudNetworkData(networkData, false),
 	)
-	return vmi, err
+	return vmi, nil
```

Comment:
Would you please remove error from the returned values, now when it's not utilized?

It should not affect anything outside the istio module.

Reply:
Done